covid_model/model_specs.py

```diff
@@ -151,6 +164,46 @@ def write_to_db(self, engine, schema='covid_model', table='specifications', tags
             session.execute(stmt)
             session.commit()
 
+    def preform_write_query(self, tags=None):
```

Comment:
these functions allow me to prepare the data to be written to the database in a picklable format, so I can fit models in parallel, then write serially to the database.

Reply:
This function prepares all the data without generating a spec_id.